lib/Rules/Lowercase.php

```diff
@@ -24,14 +24,19 @@
 class Lowercase extends Base {
 
 	/**
-	 * @param $password
-	 * @param $val
+	 * @param string $password
+	 * @param int $val
 	 * @throws PolicyException
 	 */
 	public function verify($password, $val) {
 		if ($this->countLowercase($password) < $val) {
 			throw new PolicyException(
-				$this->l10n->t('The password contains too few lowercase characters. At least %d lowercase characters are required.', [$val]));
+				$this->l10n->n(
+					'The password contains too few lowercase letters. At least one lowercase letter is required.',
+					'The password contains too few lowercase letters. At least %n lowercase letters are required.',
+					(int) $val
```

Comment:
not important, but according to phpdoc val already int, looks like no need to cast again.

Reply:
Well-spotted - that was from when I was debugging. Then at the end I made sure to always pass in ``int``

Reply:
fixed